platform/viewer/src/connectedComponents/ViewerMain.js

```diff
@@ -153,7 +152,6 @@ class ViewerMain extends Component {
             setViewportData={this.setViewportData}
           >
             {/* Children to add to each viewport that support children */}
-            <ConnectedToolContextMenu />
```

Comment:
Happy to see this go to. It does make me wonder if we need to provide a blanket way to pass children to Viewports? I'm struggling to think of a valid use case though.